pyproject.toml

```diff
@@ -54,8 +54,12 @@ a2a = [
   "httpx",
 ]
 
+composio = [
+  "composio>=0.8.9,<1"
+]
```

Comment:
Did you have a look at how heavy of a dependency this is, and whether we really want to include it in the "all"? Like we don't include the A2A dep in `all` just figured I'd mention that we should consider whether we'd like this to be included.

Reply:
It is not heavy but it is a good point, I will leave `all` reserved to "framework"